tests/src/Common/PerfHarness/PerfHarness.csproj

```diff
@@ -6,9 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="xunit.performance.api">
-      <Version>1.0.0-beta-build0006</Version>
-    </PackageReference>
+    <PackageReference Include="xunit.performance.api" Version="1.0.0-beta-build0007" />
```

Comment:
If I add `Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />` to the `PerfHarness.csproj`, then I can restore, but I cannot publish. What am I missing?

```cmd
$ "%CORECLR_ROOT%Tools\dotnetcli\dotnet.exe" --info
.NET Command Line Tools (2.0.0-preview1-005724)

Product Information:
 Version:            2.0.0-preview1-005724
 Commit SHA-1 hash:  e391b5d5f3

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.15063
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   %CORECLR_ROOT%\Tools\dotnetcli\sdk\2.0.0-preview1-005724\

$ "%CORECLR_ROOT%\Tools\dotnetcli\dotnet.exe" restore "%CORECLR_ROOT%\tests\src\Common\PerfHarness\PerfHarness.csproj"
%CORECLR_ROOT%\tests\src\Common\PerfHarness\PerfHarness.csproj
  Restoring packages for %CORECLR_ROOT%\tests\src\Common\PerfHarness\PerfHarness.csproj...
  Lock file has not changed. Skipping lock file write. Path: %CORECLR_ROOT%\tests\src\Common\PerfHarness\obj\project.assets.json
  Restore completed in 280.64 ms for %CORECLR_ROOT%\tests\src\Common\PerfHarness\PerfHarness.csproj.

  NuGet Config files used:
      %CORECLR_ROOT%\tests\src\NuGet.Config
      %APPDATA%\NuGet\NuGet.Config
      C:\Program Files (x86)\NuGet\Config\Microsoft.VisualStudio.Offline.config

  Feeds used:
      https://dotnet.myget.org/F/dotnet-core/api/v3/index.json
      https://www.myget.org/F/nugetbuild/api/v3/index.json
      https://api.nuget.org/v3/index.json

$ "%CORECLR_ROOT%\Tools\dotnetcli\dotnet.exe" publish "%CORECLR_ROOT%\tests\src\Common\PerfHarness\PerfHarness.csproj" -c Release -o "%CORECLR_ROOT%\sandbox"

Microsoft (R) Build Engine version 15.2.93.5465
Copyright (C) Microsoft Corporation. All rights reserved.

%CORECLR_ROOT%\Tools\Microsoft.CSharp.Core.targets(106,11): error MSB4064: The "OverrideToolHost" parameter is not supported by the "Csc" task. Verify the parameter exists on the task, and it is a settable public instance property. [%CORECLR_ROOT%\tests\src\Common\PerfHarness\PerfHarness.csproj]
%CORECLR_ROOT%\Tools\Microsoft.CSharp.Core.targets(67,5): error MSB4063: The "Csc" task could not be initialized with its input parameters.  [%CORECLR_ROOT%\tests\src\Common\PerfHarness\PerfHarness.csproj]
```